<launch>
  <node pkg="my_graspnet_ros" type="grasp_service_server.py" name="grasp_service_node" output="screen">
    <param name="checkpoint_path" value="$(find my_graspnet_ros)/graspnet_lib/logs/log_rs/checkpoint.tar"/>
    <param name="num_point" value="20000"/>
    <param name="num_view" value="300"/>
    <param name="collision_thresh" value="0.01"/>
    <param name="voxel_size" value="0.01"/>
    <param name="data_dir" value="$(find my_graspnet_ros)/doc/graspnet_input"/>
    <param name="factor_depth" value="1.0"/>
    <param name="image_width" value="640.0"/>
    <param name="image_height" value="480.0"/>
  </node>
</launch>